### ISSUE TITLE
Hardcoded encoding

### ISSUE THREAD

**Issue by himsve** [BitMiracle/libtiff.net#91]
Encoding is hardcoded to "Latin1" in the lines https://github.com/BitMiracle/libtiff.net/blob/0de43f782e5fa95a80f5ca1f715774d9152be463/LibTiff/Internal/Tiff_Internal.cs#L48
and https://github.com/BitMiracle/libtiff.net/blob/0de43f782e5fa95a80f5ca1f715774d9152be463/LibTiff/Internal/Tiff_DirWrite.cs#L1055.

LibTiff is then not able to write special characters such as 'degrees' symbol in customized ASCII metadata.

**Comment by Bobrovsky** [comment]
What do you suggest? 

`Encoding.ASCII` is a 7-bit one. `Encoding.UTF8` does not look like something designed for `TiffType.ASCII`. 

[The specification](https://www.itu.int/itudoc/itu-t/com16/tiff-fx/docs/tiff6.pdf) defines ASCII type in the following way:

> 2 = ASCII 8-bit byte that contains a 7-bit ASCII code; the last byte must be NUL (binary zero).

**Comment by himsve** [comment]
Yes, your are right. 7-bit ASCII is the encoding for standard TIFF metadata.

But I recognized an encoding issue in an open GDAL metadata XML tag (https://gdal.org/drivers/raster/gtiff.html and https://www.awaresystems.be/imaging/tiff/tifftags/gdal_metadata.html). The XML default endcoding is usually UTF8.
I have used LibTiff in a GeoTiff generator, where the GeoTiffs have been published to https://github.com/osgeo/proj-data.

Anyway, I solved the problem by adding a new field in Tiff_Internal.cs:
`internal static readonly Encoding UTF8Encoding = Encoding.UTF8;`
